care/users/models.py

```diff
@@ -158,9 +168,9 @@ def has_object_update_permission(self, request):
             return True
         if not self == request.user:
             return False
-        if (request.data.get("district") or request.data.get("state")) and self.user_type >= User.TYPE_VALUE_MAP[
-            "DistrictLabAdmin"
-        ]:
+        if (
+            request.data.get("district") or request.data.get("state")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Wrong hanging indentation before block (add 4 spaces).](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5316739)

Reply:
This is an issue caused by a `pre-commit` hook probably.

Reply:
@abhiabhi94 Your pre-commit was setup, right? If yes, then this comment may be ignored.

Reply:
Yes